data.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
```

Comment:
Great job @mkaya13 🤝

- [ ] [Optional] Kindly ensure the `weight_kg` values are all boolean instead of integers.

Reply:
weight_kg should be Decimal and I implemented as Decimals